documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -299,9 +299,13 @@ the provided string must not include the zone offset (`+01:00`) or the zone ID (
 Even when they make sense for the type of your field, the time and time zone may be omitted
  (if omitted, the time zone will be interpreted as the default JVM time zone).
 
-===== Dynamic boosting
+===== Index-time boosting
 
-The `org.hibernate.search.annotations.DynamicBoost` annotation is not (and cannot be) supported with Elasticsearch, because the platform lacks per-document, index-time boosting capabilities. Static boosts (`@Boost`) are, however, supported.
+The `org.hibernate.search.annotations.DynamicBoost` annotation is not (and cannot be)
+supported with Elasticsearch, because the platform lacks per-document, index-time boosting capabilities.
+
+Static boosts (`@Boost`) are supported, but as in Lucene embedded mode,
```

Comment:
> I don't think it's clear what "static boosts" mean

It's the exact same wording as in the "mapping" section.

>  think we should stop highlighting what actually works in Elasticsearch

In general I'd say yes, but here the point was that `@Boost` and `@DynamicBoost` are closely related, so I didn't want one to be confused with the other.

> Or maybe just a simple "The `@Boost` annotation will work, but since it's deprecated we suggest to use query time boosting".

Done.